packages/tracking/src/utils.ts

```diff
@@ -0,0 +1,74 @@
+declare const window: Window & {
+  dataLayer: Record<string, unknown>[];
+  trackingOptions: { debug?: boolean }
+};
+
+export const setup = () => {
+  window.dataLayer = window.dataLayer || [];
+  window.dataLayer.push({
+    'gtm.start': Date.now(),
+    event: 'gtm.js',
+  });
+};
+
+export const setupTrackingOptions = () => {
+  window.trackingOptions = window.trackingOptions || { debug: false };
+};
+export const getScriptSource = (id: string) => {
+  const source = new URL('/gtm.js', 'https://www.googletagmanager.com');
+  source.searchParams.set('id', id);
+  return source.toString();
+};
+
+
+const dumpValue = (item: unknown) => {
+  if (item instanceof Object) {
+    console.log(JSON.stringify(item));
+    console.dir(item);
+  } else {
+    console.log(item);
+  }
+};
+
+type LoggerOptions = {
+  collapsed?: boolean;
+};
+
+const getLogger = (id: string, options: LoggerOptions = {}) => (...args: unknown[]) => {
```

Comment:
I moved it to @goodhood/helpers (see https://github.com/goodhood-eu/goodhood/pull/210 ) since it feels bad to remove all the typings. Let's create a new ticket for migrating nebenan-helpers into @goodhood/helpers. It's the first step for unifying all of our packages!